dallinger/command_line.py

```diff
@@ -836,13 +843,11 @@ def export(app, local):
 
     for table in all_tables:
         subprocess.check_call(
-            "psql -d dallinger --command=\"\\copy " + table + " to \'" +
+            "psql -d " + app_name(id) +
+            " --command=\"\\copy " + table + " to \'" +
             os.path.join(subdata_path, table) + ".csv\' csv header\"",
             shell=True)
```

Comment:
Okay, I must have gotten the escaping wrong. We need to stop using `shell=True` in other places too, so we can revisit this.